src/database/database.ts

```diff
@@ -1,233 +1,75 @@
-import { URL } from 'url';
-import * as path from 'path';
-
-import { FirebaseApp } from '../firebase-app';
-import { FirebaseDatabaseError, AppErrorCodes, FirebaseAppError } from '../utils/error';
-import { FirebaseServiceInterface, FirebaseServiceInternalsInterface } from '../firebase-service';
-import { Database } from '@firebase/database';
-
-import * as validator from '../utils/validator';
-import { AuthorizedHttpClient, HttpRequestConfig, HttpError } from '../utils/api-request';
-import { getSdkVersion } from '../utils/index';
-
-
-/**
- * Internals of a Database instance.
+/*!
+ * Copyright 2020 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
  */
-class DatabaseInternals implements FirebaseServiceInternalsInterface {
-
-  public databases: {
-    [dbUrl: string]: Database;
-  } = {};
-
-  /**
-   * Deletes the service and its associated resources.
-   *
-   * @return {Promise<()>} An empty Promise that will be fulfilled when the service is deleted.
-   */
-  public delete(): Promise<void> {
-    for (const dbUrl of Object.keys(this.databases)) {
-      const db: Database = this.databases[dbUrl];
-      db.INTERNAL.delete();
-    }
-    return Promise.resolve(undefined);
-  }
-}
+import '@firebase/database';
```

Comment:
Why was this needed?

Reply:
It's no longer needed but it might've been because I wasn't importing `@firebase/database` anymore in the `index.ts` file and I needed to perform the augmentation, so I did it mostly for the side-effects. I don't think it necessary